content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -4,19 +4,43 @@ reviewers:
+To scale and application and provide a reliable service, you need to
+understand how an application behaves when it is deployed. You can examine
+application performance in a Kubernetes cluster by examining the containers,
+[pods](/docs/user-guide/pods), [services](/docs/user-guide/services), and
+the characteristics of the overall cluster. Kubernetes provides detailed
+information about an application's resource usage at each of these levels.
+This information allows you to evaluate your application's performance and
+where bottlenecks can be removed to improve overall performance.
 
 ## Overview
 
-Kubernetes monitoring is designed to be monitoring-solution agnostic.  On many clusters, monitoring is provided via two separate pipelines: the resource metrics pipeline, and full monitoring pipeline.
-The resource metrics pipeline provides a limitted set of metrics to cluster components such as the HorizontalPodAutoscaler controller, and `kubectl top`.  These metrics are generally collected by
-[metrics-server](https://github.com/kubernetes-incubator/metrics-server), and exposed via the `metrics.k8s.io` API.  metrics-server discovers all nodes on the cluster, and queries their
-[Kubelet](/docs/admin/kubelet)s for CPU and memory usage.  The Kubelet itself fetches the data from [cAdvisor](https://github.com/google/cadvisor).
+In Kubernetes, application monitoring does not depend on a single monitoring
+solution. On new clusters, you can use two separate pipelines to collect
+monitoring statistics by default:
+
+- The **resource metrics pipeline** provides a limited set of metrics related
+  to cluster components such as the HorizontalPodAutoscaler controller, as well
+  as the `kubectl top` utility. These metrics are collected by
+  [metrics-server](https://github.com/kubernetes-incubator/metrics-server)
+  and are exposed via the `metrics.k8s.io` API. `metrics-server` discovers
+  all nodes on the cluster and queries each node's [Kubelet](/docs/admin/kubelet)
+  for CPU and memory usage. The Kubelet fetches the data from
+  [cAdvisor](https://github.com/google/cadvisor). `metrics-server` is a
+  lightweight short-term in-memory store.
+  
+
+- A **full monitoring pipeline** provides richer metrics and long-term
+  storage of data. Kubernetes does not provide a specific monitoring
+  solution, but exposes the relevant metrics via the API used by
+  `metrics-server` and also in Prometheus exposition format. You can collect
+  these metrics using a monitoring agent that is compatible with either of
+  these mechanisms. Kubernetes defines two different APIs for use by
```

Comment:
This is wrong -- monitoring agents *can't* consume these APIs, since they're intended to be implemented by the those monitoring agents themselves.  Monitoring agents should consume the Prometheus metrics, and then may choose to expose either (or both) of these two APIs to provide those metrics back to the HPA.